vpr/src/base/read_options.cpp

```diff
@@ -931,6 +931,15 @@ static argparse::ArgumentParser create_arg_parser(std::string prog_name, t_optio
         .default_value("on")
         .show_in(argparse::ShowIn::HELP_ONLY);
 
+    gen_grp.add_argument<bool, ParseOnOff>(args.allow_dangling_combinational_nodes, "--allow_dangling_combinational_nodes")
+        .help(
+            "Option to allow dangling combinational nodes in the RR graph.\n"
```

Comment:
I think you mean timing graph not RR graph.
Also, I think we should say (replace middle two lines):
"This option should normally be off, as dangling combinational nodes are unusual in the timing graph and may indicate a problem in the circuit or architecture. Unless you understand why your architecture/circuit can have valid dangling combinational nodes, this option should be off."

Reply:
Exactly, it should be timing graph, thanks for the correction.
I will also add your suggested help message.

Reply:
Done